pkg/sql/sem/tree/create.go

```diff
@@ -1703,6 +1703,17 @@ func (node *SequenceOptions) Format(ctx *FmtCtx) {
 			} else {
 				ctx.Printf("%d", *option.IntVal)
 			}
+		case SeqOptRestart:
+			ctx.WriteString(option.Name)
+			ctx.WriteByte(' ')
+			if option.OptionalWord {
+				ctx.WriteString("WITH ")
+			}
+			if ctx.flags.HasFlags(FmtHideConstants) {
+				ctx.WriteByte('_')
```

Comment:
ah! maybe we should use 0 here like everywhere else

Reply:
Fixed

Reply:
I realized that the commit need fix. With `ctx.WriteByte('0')`, restart command with literal removed format, it adds 0.
I will change soon, so please ignore this commit. thanks

Reply:
fixed